<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Integral1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>1241</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft New Tai Lue</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Integral 1</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_a">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-999999999.899999976158142</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>a =</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>σ =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> μ =</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_b">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-999999999.899999976158142</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_sigma">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>0.000001000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
    <property name="value">
     <double>0.000001000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_media">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-999999999.899999976158142</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_n">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ResultadoIntegral1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>691</width>
      <height>111</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CalcularIntegral1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Integral1.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota: entre mas alto sea n mas tardara el programa en obtener el resultado, no preocuparse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Integral1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Integral1_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>1251</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Typewriter</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Integral 2</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b0 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>bf =</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_b0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-999999999.899999976158142</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_bf">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-999999999.899999976158142</double>
    </property>
    <property name="maximum">
     <double>999999999.899999976158142</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_delta">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.020000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>delta =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CalcularIntegral2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>441</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ResultadoIntegral2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>321</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Integral2.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Integral2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>810</y>
     <width>1061</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Elaborado por Edwin Bernardo Cruz villalba, Prob y Estadistica 2CV16. PD: Evite Romperlo Profe :)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBox_a</sender>
   <signal>valueChanged(double)</signal>
   <receiver>textEdit_ResultadoIntegral1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_b</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_sigma</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_media</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_n</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_b0</sender>
   <signal>editingFinished()</signal>
   <receiver>tableWidget</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_bf</sender>
   <signal>editingFinished()</signal>
   <receiver>tableWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_delta</sender>
   <signal>editingFinished()</signal>
   <receiver>tableWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_n</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral1</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_sigma</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral1</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_a</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral1</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_b</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral1</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_media</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral1</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_b0</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral2</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_bf</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral2</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_delta</sender>
   <signal>editingFinished()</signal>
   <receiver>progressBar_Integral2</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_b0</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_bf</sender>
   <signal>editingFinished()</signal>
   <receiver>textEdit_ResultadoIntegral2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>709</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
